divisiondetector/datasets.py

```diff
@@ -7,8 +7,8 @@
 logger = logging.getLogger()
 
 class DivisionDataset(Dataset):
-    def __init__(self, label_path, img_path, window_size=(100,100,100), time_window=(1, 1), mode='ball', ball_radius=(10, 10, 10)):
-        def __getlabels(label_path, div_path):
+    def __init__(self, data_path, vol_path, label_path, window_size=(100,100,100), time_window=(1, 1), mode='ball', ball_radius=(10, 10, 10)):
```

Comment:
Looks good. Would you mind to write a quick doc string for this method. Maybe you can choose different names for data_path, vol_path, label_path. Their meaning is not 100% clear for me from the name. Both label and volumes are data :). I suggest you refere to image_paths, and use positive and negative detections where appropriate.